### PR TITLE
Support for skip .Release.Service objects

### PR DESCRIPTION
**Pull request by jcabrerizo**
Support for skip .Release.Service objects + refactor validation service logic

### CODE REVIEW

File: src/commands/validate.ts [hunk 1]
```diff
@@ -41,14 +41,15 @@ export default class Validate extends SleekCommand {
...
     helmRepo: Flags.string({description: "Helm repo of the addon", exclusive: ['file', 'helmUrl'], char:'r'}),  // construct it piecemeal
     protocol: Flags.string({description: "Protocol of the helm hosting to use", exclusive: ['file', 'helmUrl'], char:'p'}),
     version: Flags.string({description: "Version of the addon to validate", exclusive: ['file'], char:'v'}),
     addonName: Flags.string({description: "Name of the addon"}),
-    skipHooks: Flags.boolean({description: "Skip helm hooks validation", default:false}),
     extended: Flags.boolean({description: "Run extended validation", hidden: true, char:'e'}), // triggers security and extended checks. NEEDS THE CONTAINER IMAGE FOR THE ADDON
-    folder: Flags.string({description: "Path to the addon folder", hidden: true, char:'d', exclusive: ['file', 'helmUrl']}),  // internal and for partner users who would like to test against their local code
+    skipHooks: Flags.boolean({description: "Skip helm hooks validation", default:false}),
+    skipResourceService: Flags.boolean({description: "Skip .Release.Service appearances", default:false}),
```
Comment (zan-mateusz) [inline src/commands/validate.ts:52]: `skipResourceService` -> `skipReleaseService` ? Also perhaps 'occurence' is a better word in this context that 'appearance' (non-native suggestion)

File: src/commands/validate.ts [hunk 2]
```diff
@@ -69,6 +70,7 @@ export default class Validate extends SleekCommand {
       addonName = flags.addonName;
     }
     let skipHooksValidation = flags.skipHooks;
+    let skipReleaseService = flags.skipResourceService;
```
Comment (zan-mateusz) [inline src/commands/validate.ts:73]: is above is changed, this needs too :) also my intellij suggests to use const instead of let as it's never reassigned

**Review by iuliana — APPROVED** [review]
LGTM

**Review by zan-mateusz — COMMENTED** [review]
LGTM, one minor thing :)

File: src/commands/validate.ts (hunk 1 above)
Comment (jcabrerizo) [reply src/commands/validate.ts]: fixed, tks

File: src/commands/validate.ts (hunk 2 above)
Comment (jcabrerizo) [reply src/commands/validate.ts]: tks, this comment flagged other error, i missed support skipping this when the input is an input yaml file

**Review by 5herlocked — APPROVED** [review]

**Pull request merged**